resources/drive-cx.ex-driveCX/templates/config/01-DriveCX.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "Locations, Purchase Details, Employees and Surveys",
+    "description": "",
+    "data": {
+   "debug": false,
+   "incrementalOutput": false,
+   "outputBucket": "DriveCX",
+   "#api_key": "",
+   "client_id": "",
```

Comment:
Why are the parameters `#api_key` and `client_id` here and are empty? I think they should be removed.